diagnostic.sql

```diff
@@ -5,24 +5,36 @@ CREATE DATABASE "sql-diagnostic";
 
 -- Question 1. Create a table called `licensees` to hold data in columns surname, given_name, gender, height, weight, born_on, eye_color, hair_color.  See `data/licensees.csv` for example data.  Don't forget that all tables should have an id column as a unique identifier for a row.
 
--- your code here
+ CREATE TABLE licensees(
+   ID int PRIMARY KEY,
```

Comment:
Yeah, saw this after the fact. The time crunch means I just kind of shoot from the hip and then fix what I can if I have time.